hack/fetch-metal3-crds.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function copy_crd {
+  local SRC="$1"
+  local DST="$2"
+  if ! diff -Naup "$SRC" "$DST"; then
+    cp "$SRC" "$DST"
+    echo "installed CRD: $SRC => $DST"
+  fi
+}
+
+REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+tmpfile="$(mktemp tmp-crd-XXXXXXXX)"
+
+# install metal3 remediation crd
+for crd in infrastructure.cluster.x-k8s.io_metal3remediations.yaml infrastructure.cluster.x-k8s.io_metal3remediationtemplates.yaml; do
+  curl -o $tmpfile https://raw.githubusercontent.com/metal3-io/cluster-api-provider-metal3/main/config/crd/bases/${crd}
```

Comment:
Just to confirm, this is for developer use and is not part of the build process right?

Reply:
Yes, I believe this is correct.

Reply:
Yes correct, it's used by a make target, but it needs to be called manually by developers.

Similar to https://github.com/openshift/cluster-baremetal-operator/blob/master/hack/get-bmh-crd.sh